chromium_src/chrome/browser/ui/tabs/tab_strip_model.cc

```diff
@@ -85,6 +90,24 @@ void TabStripModel::AppendWebContents(WebContents* contents,
     observer.TabInsertedAt(this, contents, index, foreground);
 }
 
+void TabStripModel::InsertWebContentsAt(int index,
```

Comment:
not exactly but it's similar to AppendWebContents

We don't have things like this pulled in yet: WebContentsModalDialogManager, ForgetAllOpeners